duckduckgo/Android
# Feature/bugfix only save network images

Asana Issue URL: https://app.asana.com/0/inbox/471012189846018/558766530630931/563802899182766

## Description
Check if we can download the image before showing the context menu to download it - this will will ensure the image url is http or https only (and therefore excludes data uris)


## Steps to Test this PR:
1. Visit http://www.bigfastblog.com/embed-base64-encoded-images-inline-in-html
1. Try to long press on the "HTML" image made up of green dots. No context menu should be shown.
1. If you tried the same from `develop` the app would crash.
